src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.product-card {
+  display: block;
+  box-sizing: border-box;
+
+  width: 200px;
+  height: 408px;
+
+  background-color: var(--white);
+  border-width: 1px;
+  border-radius: 5px;
+  border-color: #f3f3f3;
+  border-style: solid;
+}
+
+.product-card__image {
+  box-sizing: border-box;
+  padding-top: 32px;
+  margin: 0 19px;
```

Comment:
instead of adding horisontal margins to each element you can add a padding (`padding: 32px 16px 16px`) to whole card.

And use margin to add vertical indent between card elements